### PR TITLE
HH-3: exo-deploy: update version numbers

### PR DESCRIPTION
**Pull request by hugoesthere**
@kevgo @trushton

### CODE REVIEW

File: exo-deploy/Dockerfile [hunk 1]
```diff
@@ -10,8 +10,10 @@ COPY ./package.json /usr/src/exo-deploy
 WORKDIR /usr/src/exo-deploy
 RUN npm install --production
 
+# Copy source code
```
Comment (kevgo) [inline exo-deploy/Dockerfile:13]: This is not a proper sentence, so don't uppercase the first character

File: exo-deploy/Dockerfile [hunk 2]
```diff
@@ -10,8 +10,10 @@ COPY ./package.json /usr/src/exo-deploy
...
 COPY . /usr/src/exo-deploy
 
+# Create directory for generating terraform scripts
```
Comment (kevgo) [inline exo-deploy/Dockerfile:16]: same here

File: exo-deploy/package.json [hunk 3]
```diff
@@ -4,20 +4,20 @@
...
   "dependencies": {
     "async": "2.1.4",
-    "aws-sdk": "2.7.10",
+    "aws-sdk": "2.7.27",
     "chalk": "1.1.3",
     "dashify": "0.2.2",
-    "fs-extra": "1.0.0",
-    "handlebars": "4.0.5",
-    "inquirer": "1.2.3",
+    "fs-extra": "2.0.0",
```
Comment (kevgo) [inline exo-deploy/package.json:10]: This is a major version bump, are you sure it works correctly?

File: exo-deploy/package.json [hunk 4]
```diff
@@ -4,20 +4,20 @@
...
+    "handlebars": "4.0.6",
+    "inquirer": "3.0.1",
```
Comment (kevgo) [inline exo-deploy/package.json:12]: This is a bump of 2 major versions. I know the jump from 1 to 2 had some breaking changes. Are you sure this works still?

Comment (hugoesthere) [reply exo-deploy/package.json:12]: I have been deploying from this branch and I haven't run into any problems. I also just checked the changelogs for `fs-extra` and `inquirer` and none of the breaking changes affect us.

**Review by kevgo — APPROVED** [review]

**Pull request closed without merging**